core/domain/user_jobs_continuous.py

```diff
@@ -317,12 +322,10 @@ def map(item):
         total_rating = 0
         for ratings_value in item.ratings:
             total_rating += item.ratings[ratings_value] * int(ratings_value)
-        if not sum(item.ratings.itervalues()):
-            return
-        average_rating = total_rating / sum(item.ratings.itervalues())
+        average_rating = (
+            total_rating / sum(item.ratings.itervalues())
```

Comment:
Not sure it's needed to calculate the sum twice. Maybe calculate it just once, store it in sum_of_ratings, and refer to it in this line and the next?